doc/tutorials/klondike/step2.md

```diff
@@ -0,0 +1,258 @@
+# 2. Scaffolding
+
+In this section we will use broad strokes in order to outline the main elements
+of the game. This includes the main game class, and the general layout.
+
+## KlondikeGame
```

Comment:
The two empty lines before new sections, does that only apply to sections on the same level?

Reply:
It should be for all headers; I missed this one.